README.md

```diff
@@ -1,23 +1,67 @@
 # eris ![Logo][eris-logo]
+
 [![GoDoc][doc-img]][doc] [![Build][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![GoReport][report-img]][report] [![Discord][chat-img]][chat] [![Mentioned in Awesome Go][awesome-img]][awesome]
 
-Package eris provides a better way to handle, trace, and log errors in Go. This package is inspired by a few existing packages: [xerrors](https://github.com/golang/xerrors), [pkg/errors](https://github.com/pkg/errors), and [Go 1.13 errors](https://golang.org/pkg/errors/).
+Package `eris` provides a better way to handle, trace, and log errors in Go.
 
 `go get github.com/rotisserie/eris`
 
-## How is eris different?
+## Why you'll want to switch to eris
+
+`eris` was inspired by a simple question: what if you could fix a bug without wasting time replicating the issue or digging through the code? With `eris`, you can achieve that goal by wrapping and outputting errors to your JSON logs:
```

Comment:
In addition to the standard terminal string output, with `eris`, you can achieve that goal by wrapping and outputting errors to your JSON logs:

Basically, I like the JSON part but also want to convey that standars command line output will also work

Reply:
I'm not sure anyone cares about the string part honestly but I'll think about it